CMP1903_A1_2324/Program.cs

```diff
@@ -4,7 +4,31 @@ namespace Dice2
 
 class Program
 {
-        static void Main(string[] args);
+    static void Main(string[] args)
 
-   
+    {
+        try
+        {
+            //RunTests();
+            Game Dice = new Game();
+            Dice.Roll();
+        }
+        /*catch(InvalidDiceException e)
+        {
+            Console.WriteLine($"There is an issue {e}");
+        }*/
+        catch (Exception e)
+        {
+            Console.WriteLine(e.Message);
+        }
+
+        Console.ReadKey();
+    }
+
+    private static void RunTests()
```

Comment:
This is the definition of a method but it's never called in the program. This means that the testing class is not yet instantiated in this version of the program. The best place to call this would probably be in line 24, above Console.ReadKey();